apps/ech.c

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include "apps.h"
+#include "progs.h"
+#include <openssl/ssl.h>
+#include <openssl/bio.h>
+#include <openssl/evp.h>
+#include <openssl/err.h>
+#include <openssl/bn.h>
+#include <openssl/pem.h>
+#include <openssl/rand.h>
+#include <openssl/hpke.h>
+
+#include <openssl/objects.h>
+#include <openssl/x509.h>
+
+#ifndef OPENSSL_NO_ECH
+
+/* size for some local crypto vars */
+# define OSSL_ECH_CRYPTO_VAR_SIZE 1024
```

Comment:
this is defined as 2048 below. normal ? 

And it is currently not used in `apps\ech.c`

Reply:
Good catch. That's no longer needed in `apps/ech.c` (with the new OSSL_ECHSTORE APIs). So deleted it from there.